shared/actions/chat2/index.tsx

```diff
@@ -2497,7 +2497,7 @@ const fetchUserEmoji = async (action: Chat2Gen.FetchUserEmojiPayload) => {
       opts: {
         getAliases: true,
         getCreationInfo: false,
-        onlyInTeam: false,
+        onlyInTeam: onlyInTeam ?? false,
```

Comment:
This goes to redux. Is it gonna affect other places that use this?

Reply:
It always gets refreshed with the latest stuff when opening it.